### PR TITLE
added AUTO_INCREMENT to file V0001__R001_Create_schema.sql

### PR DESCRIPTION
**Pull request by sroeger**
Fixes Issue #423

### CODE REVIEW

File: samples/core/src/main/resources/db/migration/h2/V0001__R001_Create_schema.sql [hunk 1]
```diff
@@ -36,13 +36,13 @@ CREATE TABLE PRODUCT_AUD(
     pictureId BIGINT,
     alcoholic BOOLEAN,
     dtype VARCHAR(31) NOT NULL,
-    id BIGINT NOT NULL,
+    id BIGINT NOT NULL AUTO_INCREMENT,
```
Comment (hohwille) [inline samples/core/src/main/resources/db/migration/h2/V0001__R001_Create_schema.sql]: *_AUD tables are for envers (to save historic revisions). In *_AUD tables there can never be added a new id so there should also not be an AUTO_INCREMENT here.

**Comment by hohwille** [comment]
:+1: thanks for the fix. Could you remove the single occurrence I commented in the PRODUCT_AUD table, please. Then I am happy to merge the PR.

**Comment by sroeger** [comment]
Got it, should be fine now.

**Comment by jomora** [comment]
@hohwille, @jhcore Hey, this bug has been resolved. Could you please check and accept the PR?
Cheers!

**Comment by jhcore** [comment]
Can you confirm me if you did a merge with the current version?? To make sure we are doing the merge correctly. Can you do it? for make sure.

**Comment by sroeger** [comment]
@jhcore We just merged it with the current version, please check the changes.

**Comment by jhcore** [comment]
@sroeger done!

**Pull request merged**